### PR TITLE
login and user view models and strings

### PR DESCRIPTION
**Pull request by damfinkel**
@guidomb 
## Sumary

**Trello card**: https://trello.com/c/vVMNfPvA/10-funcionalidad-login

Login  and User view models and and some strings
## Test Plan

No visual updates
## Screenshots

No visual updates
## Known issues

Tests pending

### CODE REVIEW

File: WLXQuickBloxUtils-sample/WLXQuickBloxUtils-sample/Foundation categories/NSString+capitalizeFirstWord.m [hunk 1]
```diff
@@ -0,0 +1,20 @@
+//
+//  NSString+capitalizeFirstWord.m
+//  ToBuy
+//
+//  Created by Damian on 1/5/15.
+//  Copyright (c) 2015 wolox. All rights reserved.
+//
+
+#import "NSString+capitalizeFirstWord.h"
```
Comment (guidomb) [inline WLXQuickBloxUtils-sample/WLXQuickBloxUtils-sample/Foundation categories/NSString+capitalizeFirstWord.m]: NSString+CapitalizedFirstWord

File: WLXQuickBloxUtils-sample/WLXQuickBloxUtils-sample/ViewModels/LoginViewModel.h [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+#import "UserViewModel.h"
+
+@interface LoginViewModel : NSObject
+
+@property (readonly, nonatomic) NSString *emailPlaceholder;
+@property (readonly, nonatomic) NSString *passwordPlaceholder;
+@property (readonly, nonatomic) NSString *loginButtonTitle;
+@property (readonly, nonatomic) NSString *signUpButtonTitle;
+
+- (instancetype)initWithQuickbloxUtils:(WLXQuickBloxUtils *)quickbloxUtils;
```
Comment (guidomb) [inline WLXQuickBloxUtils-sample/WLXQuickBloxUtils-sample/ViewModels/LoginViewModel.h]: Email and password should be view model's properties.

File: WLXQuickBloxUtils-sample/WLXQuickBloxUtils-sample/Foundation categories/NSString+CapitalizeFirstWord.h [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+//
+//  Created by Damian on 1/5/15.
+//  Copyright (c) 2015 wolox. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface NSString (CapitalizeFirstWord)
+
+- (NSString*)CapitalizeFirstWord;
```
Comment (pablog) [inline WLXQuickBloxUtils-sample/WLXQuickBloxUtils-sample/Foundation categories/NSString+CapitalizeFirstWord.h]: Method's name must be camelcase.

File: WLXQuickBloxUtils-sample/WLXQuickBloxUtils-sample/Localizable.strings [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  WLXQuickBloxUtils-sample
+
+  Created by Damian on 7/10/15.
+  Copyright (c) 2015 Wolox. All rights reserved.
+*/
+
+"email" = "email";
+"password" = "password";
+"login_error" = "hubo un error. Intente nuevamente más tarde.";
+"signup_error" = "hubo un error creando la cuenta. Intente nuevamente más.";
```
Comment (pablog) [inline WLXQuickBloxUtils-sample/WLXQuickBloxUtils-sample/Localizable.strings]: "más tarde"?

**Pull request closed without merging**